scanpy/ann_data.py

```diff
@@ -91,21 +92,20 @@ def _check_dimensions(data, smp, var):
                          .format(nr_var, var.shape[0]))
 
 class AnnData(IndexMixin):
-    def __init__(self, X, smp=None, var=None, vis=None, **meta):
-        u"""
-        Annotated Matrix
+    def __init__(self, ddata=None, X=None, smp=None, var=None, vis=None, **meta):
+        """
+        Annotated Data
 
-        Represents a sample × variable matrix (e.g. cell × gene) with the
-        possibility to store an arbitrary number of annotations for both
-        samples and variables.
+        Stores a data matrix X of dimensions nr_samples x nr_variables,
```

Comment:
why did you ramove the “×” symbols? isn’t it clearer that way? i always need a moment to process that some “x” isn’t actually an x, but an ASCII approximation to “×”